pyproject.toml

```diff
@@ -48,7 +48,7 @@ dependencies = [
   "fsspec-xrootd>=0.2.3",
   "matplotlib>=3",
   "numba>=0.58.1",
-  "numpy<2,>=1.22.0",
+  "numpy>=2",
```

Comment:
This is going to be harsh for a while, any package setting `<2` won't be able to install. Also, if you do set this, you can then set new limits for everything else based on when they added numpy 2 support, like numba 0.60+. Though maybe this is jus testing?

Reply:
I found a better workaround that relaxes this.